### PR TITLE
OSDOCS-9882 notable technical changes for power vs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.16

Issue: [OSDOCS-9882](https://issues.redhat.com/browse/OSDOCS-9882)

[Link to docs preview](https://78063--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-notable-technical-changes_release-notes)

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1372,12 +1372,28 @@ In {product-title} {product-version}, {olmv1-first} changes its default upgrade
 For more information, see xref:../release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-olmv1-legacy-olm-upgrade-edges_release-notes[Support for {olmv0} upgrade edges in {olmv1} (Technology Preview)].
 
 [discrete]
-[id="ocp-4-16-builder-sa"]
-=== The builder service account is no longer created if the Build cluster capability is disabled
+[id="ocp-4-16-dal12-region_{context}"]
+=== Dal12 region added
 
-With this release, if you disable the `Build` cluster capability, the `builder` service account and its corresponding secrets are no longer created.
+With this release, the `dal12` region has been added to the Power VS Installer.
```
Comment (mramendi) [inline release_notes/ocp-4-16-release-notes.adoc:1378]: I would use "was added"

Comment (mramendi) [reply release_notes/ocp-4-16-release-notes.adoc:1378]: Also please double-check if "Power VS installer" is the right term here, as opposed to any of the existing attributes around "IBM Power".

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -1372,12 +1372,28 @@ In {product-title} {product-version}, {olmv1-first} changes its default upgrade
...
+[id="ocp-4-16-power-vs-regions_{context}"]
+=== Regions added to power virtual server
+
+This release introduces the ability to deploy to new Power virtual server (VS) regions `osa21`, `syd04`, `lon06`, and `sao01`.
+
+[discrete]
+[id="ocp-4-16-power-vs-capi_{context}"]
+=== Power virtual server cluster API updated to 0.8.0
+
+With this release, the PowerVS CAPI has been updated to version 0.8.0.
```
Comment (mramendi) [inline release_notes/ocp-4-16-release-notes.adoc:1390]: Is it correct that you have "Power VS" elsewhere but "PowerVS" here?

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -1372,12 +1372,28 @@ In {product-title} {product-version}, {olmv1-first} changes its default upgrade
...
+
+[discrete]
+[id="ocp-4-16-debug-guid_{context}"]
+=== Additional debugging statements for ServiceInstanceNameToGUID
+
+With this release, additional debugging statements have been added to the `ServiceInstanceNameToGUID` function.
```
Comment (mramendi) [inline release_notes/ocp-4-16-release-notes.adoc:1396]: I would use "were added". The tense advise in IBM SG is to keep it as simple as possible https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (mramendi) [reply release_notes/ocp-4-16-release-notes.adoc:1378]: Also if the region is verbosely "dal12" then maybe don't capitalize the title?

**Review by mramendi — APPROVED** [review]
Looks great for peer review, my comments are really nitpicking.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -1379,6 +1379,30 @@ With this release, if you disable the `Build` cluster capability, the `builder`
...
+[id="ocp-4-16-dal12-region_{context}"]
+=== dal12 region was added
+
+With this release, the `dal12` region has been added to the {ibm-power-name} VS Installer.
+
+[discrete]
+[id="ocp-4-16-power-vs-regions_{context}"]
+=== Regions added to power virtual server
+
+This release introduces the ability to deploy to new Power virtual server (VS) regions `osa21`, `syd04`, `lon06`, and `sao01`.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-16-release-notes.adoc:1392]:
```suggestion
This release introduces the ability to deploy to new Power Virtual Server (VS) regions `osa21`, `syd04`, `lon06`, and `sao01`.
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -1379,6 +1379,30 @@ With this release, if you disable the `Build` cluster capability, the `builder`
...
 
+[discrete]
+[id="ocp-4-16-dal12-region_{context}"]
+=== dal12 region was added
+
+With this release, the `dal12` region has been added to the {ibm-power-name} VS Installer.
+
+[discrete]
+[id="ocp-4-16-power-vs-regions_{context}"]
+=== Regions added to power virtual server
```
Comment (JoeAldinger) [inline release_notes/ocp-4-16-release-notes.adoc:1390]:
```suggestion
=== Regions added to Power Virtual Server
```

Comment (JoeAldinger) [reply release_notes/ocp-4-16-release-notes.adoc:1390]: I couldn't find anywhere in the repo where this wasn't capitalized.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 6]
```diff
@@ -1379,6 +1379,30 @@ With this release, if you disable the `Build` cluster capability, the `builder`
...
+
+This release introduces the ability to deploy to new Power virtual server (VS) regions `osa21`, `syd04`, `lon06`, and `sao01`.
+
+[discrete]
+[id="ocp-4-16-power-vs-capi_{context}"]
+=== Power virtual server cluster API updated to 0.8.0
```
Comment (JoeAldinger) [inline release_notes/ocp-4-16-release-notes.adoc:1396]:
```suggestion
=== Power Virtual Server cluster API updated to 0.8.0
```

File: release_notes/ocp-4-16-release-notes.adoc (hunk 5 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-16-release-notes.adoc:1390]: If this is an IBM specific thing, it seems like IBM precedes Power Virtual Server `IBM Power Virtual Server`.

**Review by JoeAldinger — COMMENTED** [review]
A few capitalization suggestions. Ping me back and I'll merge.

**Comment by JoeAldinger** [comment]
Merge requested via slack.

**Pull request merged**